include/Session.h

```diff
@@ -14,12 +16,30 @@
 class User;
 class Watchable;
 
+enum ActionStringsEnum{
+    createuser,
+    changeuser,
+    deleteuser,
+    dupuser,
+    listContent,
+    watchlist,
+    watch,
+    logActions, //because log exists in <vector>.
+    exitLoop // because exit exists in <unordered_map>.
+};
+
+const std::string ACTION_PARAMS_DELIMITER = " ";
+const std::string CONTINUE = "y";
+const long NOTHING_TO_RECCOMAND = -1;
```

Comment:
NOTHING_TO_RECOMMEND*

Reply:
I knew it's a mistake and I searched it for HOURS!! thanks :sweat_smile: